### ISSUE TITLE
Error when sending emails to a new email address

### ISSUE THREAD

**Issue by silicht** [Intermesh/groupoffice#1034]
When I send an email to an address that is not stored in the address book, I get the following error popup:

```
strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated
```

The email is sent in the background, but the mail remains open after the error is accepted. This behavior suggests to the user that the mail was not sent. This causes mails to be sent multiple times.

The following message is logged on the server:

```
[Sun Jul 23 12:39:17 2023] [notice] [pid 35] [client 192.168.32.1:35130] ErrorException in /usr/local/share/groupoffice/go/base/util/StringHelper.php at line 418: strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated
```

**Comment by silicht** [comment]
The problem seems to occur when a simple mail address is used as recipient (without name in angle brackets) and the dialog "Save unknown recipients in address book" is activated.

**Comment by derjoachim** [comment]
Reproduced and fixed. To be released in the next 6.7 version, which I expect to be this week-ish.

**Comment by derjoachim** [comment]
6.7.43 has been released, so I'll close this ticket.
